bundles/GenericExecutionEngineBundle/src/Messenger/Handler/AbstractAutomationActionHandler.php

```diff
@@ -151,17 +151,47 @@ protected function getCurrentJobStepConfig(GenericExecutionEngineMessageInterfac
         return $config;
     }
 
+    protected function getEnvironmentVariables(
+        GenericExecutionEngineMessageInterface $message
+    ): array {
+        $jobRun = $this->getJobRun($message);
+        $job = $jobRun->getJob();
+
+        return $job === null ? [] : $job->getEnvironmentData();
+    }
+
+    protected function replaceConfigValueWithEnvVariable(
+        string $value,
+        array $variables
+    ): mixed {
+        /** @var $matches array */
+        if(!preg_match_all("/pimcore_copilot_get_env\('([^']*)'\)/", $value, $matches)) {
```

Comment:
I think it is weird that the Core refers to things from an extension.

Reply:
For sure, you are absolutely right, the name should reflect the generic execution engine.